lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -545,6 +545,7 @@ def find_world_ip
     def dump_config
       config = [
         'Setup configuration:',
+        "   VM name .................... CL#{@options['client_id']}",
```

Comment:
```Studio Properties:
Setup configuration:
VM name .................... CL0
```

As we have a `Studio Properties` we can skip `VM name` line, also `CL0` is not so clear.